packaging/installer/kickstart.sh

```diff
@@ -411,18 +411,23 @@ cleanup() {
 
 deferred_warnings() {
   if [ -n "${NETDATA_WARNINGS}" ]; then
+    troubleshoot_guide_url="https://learn.netdata.cloud/docs/installing/installation-troubleshoot-actions"
     printf >&2 "%s\n" "The following non-fatal warnings or errors were encountered:"
     # shellcheck disable=SC2059
     printf >&2 "${NETDATA_WARNINGS}"
+    printf >&2 "\n"
+    printf >&2 "%s\n" "You can consult the guide to common installation problems in our documentation ${troubleshoot_guide_url}"
     printf >&2 "\n\n"
   fi
 }
 
 fatal() {
+  troubleshoot_guide_url="https://learn.netdata.cloud/docs/installing/installation-troubleshoot-actions"
   deferred_warnings
   printf >&2 "%s\n\n" "${TPUT_BGRED}${TPUT_WHITE}${TPUT_BOLD} ABORTED ${TPUT_RESET} ${1}"
   printf >&2 "%s\n" "For community support, you can connect with us on:"
   support_list
+  printf >&2 "%s\n" "Or you can consult the guide to common installation problems in our documentation ${troubleshoot_guide_url}"
```

Comment:
I would suggest putting this above the community support list. We also need a variable to ensure that `deferred_warnings()` doesn’t print this if called by `fatal()` (otherwise it just looks bad).